test/libyul/yulSyntaxTests/eof/object_name_in_eof.yul

```diff
@@ -0,0 +1,9 @@
+object "a.b" {
+    code {}
+}
+
+// ====
+// EVMVersion: >=shanghai
+// bytecodeFormat: >=EOFv1
+// ----
+// SyntaxError 9822: In EOF context object name "a.b" must not contain 'dot' character.
```

Comment:
I think you forgot to update this test. It does not show a source location.

Reply:
Updated but it shows now location of the first block so for this case location of the curly brackets of `code`. I think there is no easy way to fix it. FIXME added.

Reply:
That's fine. I expected it to work exactly like that. The message includes the object name so it should be easy to figure out.

A FIXME is a good idea too. I don't think there's anything we can do without changing how we represent Yul ASTs internally and it would be a big change, but a comment for things like this is always good. It will look puzzling without context.